backend/src/helpers/UserHelper.js

```diff
@@ -1,43 +1,41 @@
 // Priorities are set from the lowest to the highest.
```

Comment:
UserHelper -> `helpers.js` or `utils.js`?

Reply:
Since it's not a file with a `UserHelper` class it can be renamed. `utils.js` IMO is better naming.